src/pythonjsonlogger/core.py

```diff
@@ -215,6 +219,7 @@ def __init__(
         self._required_fields = self.parse()
         self._skip_fields = set(self._required_fields)
         self._skip_fields.update(self.reserved_attrs)
+        self.defaults = defaults
```

Comment:
I'm wondering if instead of this being `Optional[Dict[...]]` we instead force it to be a dict
```suggestion
        self.defaults = defaults if defaults is not None else {}
```